### DISCUSSION TITLE
How to best measure out labels. Avery 5520.

### DISCUSSION THREAD

**Discussion by LEAmberry1** [snipe/snipe-it#17613] [Q&A]
We are using Avery 5520 labels and for the life of me, I can not figure out the right measurements for this.

The barcodes are not fully in the squares and then, that renders them unscannable. They are merged into others.

Any tips?

**Comment by welcome** [comment]
👋 Thanks for opening your first issue here! If you're reporting a 🐞 bug, please make sure you include steps to reproduce it. We get a lot of issues on this repo, so please be patient and we will get back to you as soon as we can.
